test/runtime/specs/juttle-spec/stdlib/select.spec.md

```diff
@@ -179,3 +179,63 @@
     {parity:1, x: 15}
     {parity:0, x: 36}
     {parity:1, x: 35}
+
+## top selects the 5 largest values
+### Juttle
+    import "select.juttle" as select;
+    emit -limit 10 -from Date.new(0)
+    | put x = 10-count()
+    | select.top -n 5 -by 'x'
+    | keep x
+    | view result
+
+### Output
+    {x: 9}
+    {x: 8}
+    {x: 7}
+    {x: 6}
+    {x: 5}
+
+## bottom selects the 5 smallest values
+### Juttle
+    import "select.juttle" as select;
+    emit -limit 10 -from Date.new(0)
+    | put x = 10-count()
+    | select.bottom -n 5 -by 'x'
+    | keep x
+    | view result
+
+### Output
+    {x: 4}
+    {x: 3}
+    {x: 2}
+    {x: 1}
+    {x: 0}
+
+## top selects the 5 largest values from 3 element stream
+### Juttle
+    import "select.juttle" as select;
+    emit -limit 3 -from Date.new(0)
+    | put x = 3-count()
+    | select.top -n 5 -by 'x'
+    | keep x
+    | view result
+
+### Output
+    {x: 2}
+    {x: 1}
+    {x: 0}
+
+## bottom selects the 5 smallest values from 3 element stream
+### Juttle
+    import "select.juttle" as select;
+    emit -limit 3 -from Date.new(0)
+    | put x = 3-count()
+    | select.bottom -n 5 -by 'x'
+    | keep x
+    | view result
+
+### Output
```

Comment:
Intuitively I want to go the other way for `bottom`, so smallest value is first, otherwise I think I'd just have to `select.bottom | sort -asc` every time. Are we going by standard expectations as in other libraries?